scripts/optimise_raw_camera.py

```diff
@@ -0,0 +1,166 @@
+#! /usr/bin/env python
+
+import os,sys,glob
+import pandas as pd
+import geopandas as gpd
+import numpy as np
+from skysat_stereo import asp_utils as asp
+from scipy.optimize import least_squares
+from pyquaternion import Quaternion
+import argparse
+from pygeotools.lib import iolib,geolib
+import logging
+
+def cam_solve(q1,q2,q3,q4,CX,CY,CZ,cu,cv,fu,fv,pitch,X,Y,Z):
```

Comment:
Great to see these modifications @ShashankBice !  Perhaps in the future, we can create a pinhole camera class with all of these attributes (and maybe relevant methods), rather than passing 12 arguments.